pkg/ext/stores/tokens/tokens.go

```diff
@@ -1022,39 +1070,46 @@ func (tp *tokenHasher) MakeAndHashSecret() (string, string, error) {
 
 // UserName hides the details of extracting a user name and its permission
 // status from the request context
-func (tp *tokenAuth) UserName(ctx context.Context, store *SystemStore) (string, bool, bool, error) {
+func (tp *tokenAuth) UserName(ctx context.Context, store *SystemStore, verb string) (string, bool, bool, error) {
 	userInfo, ok := request.UserFrom(ctx)
 	if !ok {
+		logrus.Error("ext.cattle.io/token username: context has no user info")
 		return "", false, false, apierrors.NewInternalError(fmt.Errorf("context has no user info"))
 	}
 
 	decision, _, err := store.authorizer.Authorize(ctx, &authorizer.AttributesRecord{
 		User:            userInfo,
-		Verb:            "*",
+		Verb:            verb,
 		Resource:        "*",
 		ResourceRequest: true,
 	})
 	if err != nil {
+		logrus.Errorf("ext.cattle.io/token username: authorization error: %v", err)
```

Comment:
Is there any chance of leaking to logs any sensitive information here?

Reply:
logs dropped, c68960a6eb2bf141865c3b325dc9aa35e9dae7a0